doc/DEVELOPING.md

```diff
@@ -249,13 +248,15 @@ within 24 hours. In order to preserve author information for commits, use the
 
 Nicotine+ uses the following versioning scheme: `A.B.C`, e.g. `3.3.8`
 
-  - A is incremented when major changes are made to the user interface that
+  - `A` is incremented when major changes are made to the user interface that
     significantly change the workflow of the application. In practice, this is
     unlikely to happen.
-  - B is incremented when significant features or new dependencies are added,
+  - `B` is incremented when significant features or new dependencies are added,
     version requirements are increased, or in the case of major changes to
-    existing code that are deemed too intrusive for a smaller release.
-  - C is incremented when bug fixes, performance improvements or smaller
+    existing code that are deemed too intrusive for a smaller release. Then the
+    previous feature branch (e.g. `3.3.x`) continues to be maintained only with
+    critical backports, until it's dependencies reach beyond end-of-life.
```

Comment:
Realistically, the previous branch will be maintained until the next `B` is released (3.3.x will reach EOL once 3.4.0 is released). There isn't much point in maintaining the older branch after that, since Debian-based distros will not package such releases (only major software like web browsers receive major version bumps, everything else is frozen with the exception of security backports).